### PR TITLE
Fix a typo in the docs

### PR DESCRIPTION
**Pull request by Numpsy**
Just something I noticed when reading the docs.

### CODE REVIEW

File: docs/content/Getting Started Using.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ dotnet msbuild YourSolution.sln /t:AnalyzeFSharpProject
 
 ### Select specific projects
 
-As we don't want to targt all projects of the solution, we create a second custom MSBuild target that calls the project-specific target for all relevant projects.  
+As we don't want to target all projects of the solution, we create a second custom MSBuild target that calls the project-specific target for all relevant projects.
```
Comment (Numpsy) [inline docs/content/Getting Started Using.md:122]: Whilst in the area - would 'in the solution' sound better than 'of the solution'?

Comment (baronfel) [reply docs/content/Getting Started Using.md:122]: Yeah, that reads better to my American English ears.

Comment (dawedawe) [inline docs/content/Getting Started Using.md:122]:
```suggestion
As we don't want to target all projects in the solution, we create a second custom MSBuild target that calls the project-specific target for all relevant projects.  
```

**Review by nojaf — APPROVED** [review]
Thanks

**Pull request merged**